src/renderer/components/layout/sidebar-item.tsx

```diff
@@ -6,42 +6,49 @@ import { cssNames, prevDefault } from "../../utils";
 import { observer } from "mobx-react";
 import { NavLink } from "react-router-dom";
 import { Icon } from "../icon";
-import { TabLayoutRoute } from "./tab-layout";
 import { sidebarStorage } from "./sidebar-storage";
 import { isActiveRoute } from "../../navigation";
 
 interface SidebarItemProps {
-  id: string; // Used to save nav item collapse/expand state in local storage
+  id: string; // unique id, used in storage and integration tests
   url: string;
-  text: React.ReactNode | string;
   className?: string;
+  title: React.ReactNode;
   icon?: React.ReactNode;
   isHidden?: boolean;
-  isActive?: boolean;
-  subMenus?: TabLayoutRoute[];
+  isActive?: boolean; // marks a link, by default checks `props.url` to identify active state
+  subMenu?: React.ReactNode | React.ComponentType<SidebarItemProps>[]; // props.children could be used too
```

Comment:
Why do we need this change? Isn't the problem that `isActive` wasn't computed?

Reply:
It's not related to `isActive`. `props.subMenu` has basically same role as `props.children`, could be even removed (currently both adds data to sub-menu container).

Reply:
Yeah I have noticed that they are very similar indeed. But for a bug fix I think it should be a small as possible (within reason).

Reply:
Renamed back to `props.subMenus`